handbook/technical-overview/adr/0014-logging-apm-monitoring.md

```diff
@@ -0,0 +1,98 @@
+| Development                         | 160                | 0             | 0                         | ISK3,200,000.00                   | ISK3,200,000.00                        |
+| Infrastructure                      | 0                  | 0             | \$700.00                  | ISK1,100,400.00                   | ISK2,200,800.00                        |
+| Monitoring                          | 8                  | 0             | ISK10,000.00              | ISK280,000.00                     | ISK400,000.00                          |
+| Maintenance                         | 0                  | 5             | 0                         | ISK1,200,000.00                   | ISK2,400,000.00                        |
+| Total                               |                    |               |                           | **ISK5,780,400.00**               | **ISK8,200,800.00**                    |
+|                                     |                    |               |                           |                                   |                                        |
+| **SaaS**                            |                    |               |                           |                                   |                                        |
+| DataDog | 0 | 0 | $1,750.00 | **ISK2,751,000.00** | **ISK5,502,000.00** | |
+|                                     |                    |               |                           |                                   |                                        |
+|                                     |                    |               |                           |                                   |                                        |
+|                                     |                    |               |                           |                                   |                                        |
+|                                     |                    |               |                           |                                   |                                        |
+| Sample hourly rate without VAT, ISK | ISK20,000.00       |               |                           |                                   |                                        |
+| ISK/USD                             | ISK131.00          |               |                           |                                   |                                        |
+
+## Decision Outcome
+
+SaaS solution (DataDog) is a clear winner due to its hastle-free service usage, continuous improvement as well as its competitive pricing.
+
+### Positive Consequences
+
+- Easy access to observability for devs and devops
+- Well known integration target for third-party services
+- Monitoring setup is well known in the industry
+- No special knowledge about setup of the plaftorm
+
+### Negative Consequences
+
+None.
```

Comment:
Costs a lot

Reply:
I don't think this counts as a negative consequence, especially since I have the cost estimation. Both options cost a lot. This of course is up for interpretation, but I was thinking this section would be about some negative effect on the tech or processes.

Reply:
Yeah, I misinterpreted the data in the table, by reading raw markdown 🙈.